### ISSUE TITLE
listbots does not work sometimes

### ISSUE THREAD

**Issue by sweet235** [Unvanquished/Unvanquished#2341]
When building from master and connecting a client to a local server, I observe this in the client's console:
```
]/listbots
listbots: 2 bots in game: 
Slot Name Team [s=skill ss=skillset b=behavior g=goal] 
$1$ $2$ 
$1$ $2$
```

But it works as expected on the server's console:
```
]/listbots
listbots: 2 bots in game: 
Slot Name Team [s=skill ss=skillset b=behavior g=goal] 
1 [bot] Lena: alien [s=5 ss="feels-pain mara-flee-jump tyrant-flee-run mantis-flee-jump fighting movement goon-flee-jump goon-attack-jump mara-attack-jump tyrant-attack-run mantis-attack-jump" b=default g=(2629 1723 130)] 
2 [bot] Audran: human [s=5 ss="fighting predict-aim movement feels-pain buy-armor prefer-armor" b=default g=(team_human_mgturret|#73)]
```

**Comment by DolceTriade** [comment]
Anything more specific? Maybe an escaping issue?

**Comment by sweet235** [comment]
It would be easy to fix, but I don't know why it was rewritten. So I better not touch it.

**Comment by necessarily-equal** [comment]
The reason for that, is to avoid a buffer overflow when there are many bots. See https://github.com/Unvanquished/Unvanquished/pull/2269#discussion_r1006343570 and https://github.com/Unvanquished/Unvanquished/pull/2269#discussion_r1008813638.

**Comment by illwieckz** [comment]
Was it fixed by recent list* rework?

**Comment by slipher** [comment]
Fixed by #2384.
